Oracle/lab_result_cm.sql

```diff
@@ -114,48 +118,62 @@ and (lab.LAB_ORDER_DATE - norm.birth_date) > norm.age_lower
 and (lab.LAB_ORDER_DATE - norm.birth_date) <= norm.age_upper
 /
 
+create table lab_result_w_source as
+select lab.*
+, NVL(code, 'NI') SPECIMEN_SOURCE
+from lab_result_w_norm lab
+left join
+(
+  select distinct instance_num, NVL(code, 'OT') code
+  from &&i2b2_data_schema.supplemental_fact sf
+  left join specimen_source_map ssm on lower(sf.tval_char) = lower(ssm.specimen_source_name)
+  and ssm.specimen_source_name is not null
```

Comment:
As I left multiple nulls in the SPECIMEN_SOURCE_NAME column of the specimen_source_map.csv, I'm explicitly excluding nulls here.  Not sure I'm happy with this approach.